shop/src/utils/generateVariants.js

```diff
@@ -40,6 +40,10 @@ export const generateVariants = (product) => {
     (optionCombo, index) => {
       const comboTitle = `${product.title} - ${optionCombo.join(' / ')}`
 
+      const prevValues = {
+        ...existingVariants[optionCombo.join('|||')]
```

Comment:
I hadn't notice we use this interesting separator before :)
Maybe add a comment about why we need such a atypical separator?
Also where is this used? I did not see any reference to a split by '|||' anywhere in the dshop code?

Reply:
I just used that as a form of serialization, we didn't want to lose old values when a variant is added/deleted. Didn't want to complicate things, so went with a simple string concat. 

It's set and used in the same file: https://github.com/oplabs/dshop/blob/shahthepro/fix-883/shop/src/utils/generateVariants.js#L35

Will add a comment.